servers/visual/rasterizer.h

```diff
@@ -930,6 +930,7 @@ class Rasterizer {
 	virtual RasterizerScene *get_scene()=0;
 
 	virtual void initialize()=0;
+	virtual void update_system_fbo(uint32_t p_system_fbo)=0;
```

Comment:
This is kind of awkward, but I understand that It's difficult to pass given VisualServer creates it.
To avoid polluting this class, which is unrelated to OpenGL, how about a static GLuint in RasterizerStorageGLES3 ? it defaults to 0, but on IOS, in os_ios.cpp  you can do:

RasterizerStorageGLES3::system_fbo=<applefbo>